app/controllers/idv/review_controller.rb

```diff
@@ -102,13 +102,15 @@ def address_verification_method
     def init_profile
       idv_session.create_profile_from_applicant_with_password(password)
 
+      gpo_mail = GpoMail.new(current_user)
```

Comment:
`gpo_mail_service` in this controller is created on line 44, but since we are reusing it we could possibly extract it into its own method like in the `gpo_controller`. It is also namespaced to `Idv::GpoMail.new(...)`